tools/isolab/isolab.xml

```diff
@@ -0,0 +1,587 @@
+<!--
+  wl-11-04-2018, wed: commence
+  wl-12-04-2018, Thu: first working draft
+  wl-13-04-2018, Fri: enter target information manually. The main 
+     reference is galaxy tool 'masigpro'.
+  wl-05-02-2019, Tue: add load group file and help   
+  wl-24-03-2019, Sun: R package 'ecipex' is not in bioconda and conda-forge
+  wl-02-04-2019, Tue: add more tests and use 'sed' to remove the first and 
+   bottom lines of '$target_matrix'. Note that sed's '$' need to escape.
+-->
+
+<tool id="isolab" name="IsotopicLabelling" version="0.1.0">
+  <description>
+    Mass Spectrometric Isotopic Labelling
+  </description>
+  <macros>
+    <import>macros.xml</import>
+  </macros>
+  <expand macro="requirements" />
+  <expand macro="stdio" />
+  
+  <!-- =============================================================== -->
+  <command>
+    <![CDATA[
+      #if str($target.target_selector) == "no":
+          sed -i '1d;\$d' '$target_matrix' &&
+          #if $target.target_output:
+              ln -f $target_matrix $target_out &&
```

Comment:
please use single quotes surrounding all data and text params and configfiles

Reply:
In this case you would like to use cp instead of ln ... as the target_matrix will be deleted when the job has finished.